src/index.js

```diff
@@ -1,16 +1,13 @@
-// react libraries
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { Provider} from 'react-redux';
-import configureStore from '../src/store/configureStore'
 
-// components
-import Router from './components/Router';
+// IMPORT STYLESHEET
```

Comment:
#### Critical
Please remove unneeded lines of code instead of commenting them out.

Reply:
fixed